Configuration/Settings.Neos.yaml

```diff
@@ -2,6 +2,4 @@ Neos:
   Flow:
     mvc:
       routes:
-        PunktDe.OAuth2.Server:
```

Comment:
Correct (as the package does not require Neos), but breaking as AFAIK it won't work like this when Neos is installed. We should mention that in the Readme.

Reply:
@daniellienert Or add "position: start"?

Reply:
Even better!!